apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/entity/ServerConfig.java

```diff
@@ -42,7 +42,7 @@ public class ServerConfig extends BaseEntity {
   @Column(name = "`Value`", nullable = false)
   private String value;
 
-  @Column(name = "Comment", nullable = false)
+  @Column(name = "`Comment`", nullable = false)
```

Comment:
I've noticed that some columns are escaped and some are not. Is there a pattern behind this? One problem is that future changes that add new columns may miss escaping.

Reply:
@nobodyiam I use **"Comment"** in my init sql. It requires escaped on column. So far, other not escaped colums has no matter. But I would like to fill them in another PR.

> One problem is that future changes that add new columns may miss escaping.
I think we can try to find how to lint this or we can ask contributors must add escapsed on table、column define

Reply:
> I use "Comment" in my init sql. It requires escaped on column.

I'm still curious about why the `Comment` column needs to be escaped when it's used in init sql. Is there any error occurring if not escaped?

Reply:
@nobodyiam If we don't escapsed it, it will be **comment** or **COMMENT** in h2 database.(default is **COMMENT, you can change it to comment in h2 url)